### DISCUSSION TITLE
how do i get the app to run?

### DISCUSSION THREAD

**Discussion by lashmanRB** [flet-dev/flet#735] [Q&A]
either i'm doing something wrong, or something's broken

i'm using Visual Studio Code, and after installing flet, i try to run the example app from the docs, but after running the file via python it just doesn't do anything, i'm not getting any errors, it just goes back to the prompt, doesn't even display any windows. i tried both the standalone and web versions, neither work. 

what am i doing wrong? i tried running it on python 3.9, 3.10, and 3.11

**Comment by ndonkoHenri** [comment]
Strange. 🤔

**Comment by ndonkoHenri** [comment]
Can you try creating a new venv and installing just flet?
